packages/eslint-plugin/docs/rules/no-confusing-void-expression.md

```diff
@@ -109,6 +109,29 @@ function doSomething() {
 console.log(void alert('Hello, world!'));
 ```
 
+### `ignoreVoidInVoid`
+
+Allow using `void` type expressions in return value of a function that specified as `void`
+
+Examples of additional **correct** code with this option enabled:
+
+```ts option='{ "ignoreVoidInVoid": true }' showPlaygroundButton
+function returnsVoid(): void {}
```

Comment:
[docs] It seems that other examples in the rules documentation use `window.postMessage(value)` as an example of a function that returns `void`. What do you think about using this function instead of `returnsVoid`?

Reply:
that's make sence. i edit the docs using `window.postMessage(value)`